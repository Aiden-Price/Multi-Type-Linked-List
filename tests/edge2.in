NEW 3
start
middle
end
INSERT 0 0 first
DELETE 0 0