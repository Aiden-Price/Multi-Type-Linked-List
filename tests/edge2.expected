List 0: start -> middle -> end
List 0: first -> start -> middle -> end
